modules/javafx.controls/src/test/java/test/javafx/scene/control/TableViewRowTest.java

```diff
@@ -205,4 +207,30 @@ protected boolean isItemChanged(String oldItem, String newItem) {
 
         assertTrue(isItemChangedCalled.get());
     }
+
+    @Test
+    void testUpdateRowIndexManually() {
+        TableView<String> table = ControlUtils.createTableView();
+
+        TableRow<String> row = new TableRow<>();
+        row.updateTableView(table);
+
+        stageLoader = new StageLoader(row);
+
+        row.updateIndex(0);
+
+        List<TableCell<String, String>> cells = row.getChildrenUnmodifiable().stream().
```

Comment:
minor suggestion: maybe reformat to keep one statement per line

```
row.getChildrenUnmodifiable().stream().
   filter(TableCell.class::isInstance).
   map(e -> (TableCell<String, String>) e).
   toList();
```

Reply:
(here and elsewhere)

Reply:
I usually keep it this way as it is still readable and less space. But changed the formatting so that the statement with the dot `.filter` is in the new line. This is as far as I know what most people do.

Reply:
there is generally added benefit in case of an exception the line number would point to the offending code, rather than to the whole line.

it's probably not important here as we don't deal with deciphering ancient logs as much.